src/test/java/com/softserveinc/ita/mmakoviichuk/RozetkaTest.java

```diff
@@ -2,46 +2,53 @@
 
 import com.softserveinc.ita.mmakoviichuk.pageobjects.rozetka.*;
 import com.softserveinc.ita.mmakoviichuk.utils.runners.TestRunner;
-import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-public class RozetkaTest extends TestRunner {
-
-    @Test
-    public void categoryTest() {
-        HomePage homePage = new HomePage();
-        String categoryUrl = homePage.getCategoryUrl(1);
-        homePage.openCategory(1);
-        Assert.assertEquals(categoryUrl, homePage.getCurrentUrl(categoryUrl));
-    }
+import static org.assertj.core.api.Assertions.assertThat;
 
-    @Test
-    public void dropdownCategoryTest() {
-        HomePage homePage = new HomePage();
-        String categoryUrl = homePage.getDropdownCategoryUrl(1);
-        homePage.openCategoryFromDropdown(1);
-        Assert.assertEquals(categoryUrl, homePage.getCurrentUrl(categoryUrl));
-    }
+public class RozetkaTest extends TestRunner {
 
     @DataProvider
-    public Object[][] rozetkaLoginData() {
+    public Object[][] rozetkaCategoryData() {
         return new Object[][]{
-                {"dospecwork@gmail.com", "Qwerty123",}};
+                {"Товари для геймерів"},
+                {"Побутова техніка"},
+                {"Товари для дому"}
+        };
     }
-
-    @Test(dataProvider = "rozetkaLoginData")
-    public void wishlistTest(String email, String password) {
+    @Test(dataProvider = "rozetkaCategoryData")
+    public void categoryTest(String title) {
         HomePage homePage = new HomePage();
-        homePage.logIn(email, password);
-        getDriver().get("https://rozetka.com.ua/ua/41556706/g41556706/");
-
-        ProductPage productPage = new ProductPage();
-        productPage.addToWishlist();
-        String id = productPage.getProductId();
-        WishlistPage wishlistPage = homePage.openWishList();
+        homePage.openCategory(title);
+        assertThat(title).contains(new CategoryPage().getCategoryTitle());
+    }
 
-        boolean isProductInWishlist = wishlistPage.isContainsProductId(id);
-        Assert.assertTrue(isProductInWishlist);
+    @Test(dataProvider = "rozetkaCategoryData")
+    public void dropdownCategoryTest(String title) {
+        HomePage homePage = new HomePage();
+        homePage.openCategoryFromDropdown(title);
+        assertThat(title).contains(new CategoryPage().getCategoryTitle());
     }
+//    Needs cookie(will implement later)
```

Comment:
no commented tests in the project, take a look @Disabled annotation in testng, and place it upper of @Test annotation